tests/unit/selectmenu/core.js

```diff
@@ -341,23 +341,40 @@ $.each( [
 	} );
 } );
 
-	QUnit.test( "Selectmenu should reset when its parent form resets", function( assert ) {
-		var ready = assert.async();
-		assert.expect( 2 );
+QUnit.test( "Selectmenu should reset when its parent form resets", function( assert ) {
+	var ready = assert.async();
+	assert.expect( 2 );
 
-		var element = $( "#speed" ).selectmenu(),
-			widget = element.selectmenu( "widget" ),
-			initialValue = element.val(),
-			form = element.closest( "form" );
+	var element = $( "#speed" ).selectmenu(),
+		widget = element.selectmenu( "widget" ),
+		initialValue = element.val(),
+		form = element.closest( "form" );
 
-		element.val( "Slower" );
-		element.selectmenu( "refresh" );
-		assert.equal( $.trim( widget.text() ), "Slower" );
-		form[ 0 ].reset();
-		setTimeout( function() {
-			assert.equal( $.trim( widget.text() ), initialValue );
-			ready();
-		} );
+	element.val( "Slower" );
+	element.selectmenu( "refresh" );
+	assert.equal( $.trim( widget.text() ), "Slower" );
+	form[ 0 ].reset();
+	setTimeout( function() {
+		assert.equal( $.trim( widget.text() ), initialValue );
+		ready();
 	} );
+} );
+
+// http://bugs.jqueryui.com/ticket/15031
+QUnit.test( "#15031: Number pad input should change value", function( assert ) {
```

Comment:
I've seen both (the prefix in the test title and the comment before the actual test) in the current code base. Which one are you talking about?